metrics-influxdb/src/test/java/com/izettle/metrics/influxdb/InfluxDbReporterTest.java

```diff
@@ -235,6 +235,56 @@ public void reportsIncludedMeters() throws Exception {
     }
 
     @Test
+    public void shouldMapMeasurementToDefinedMeasurementNameAndRegex() {
+        Map<String, String> measurementMappings = new HashMap<String, String>();
+        measurementMappings.put("resources", ".*resources.*");
+
+        InfluxDbReporter filteredReporter = InfluxDbReporter
+            .forRegistry(registry)
+            .measurementMappings(measurementMappings)
+            .build(influxDb);
+
+        filteredReporter.report(
+            this.<Gauge>map(),
+            this.<Counter>map(),
+            this.<Histogram>map(),
+            this.map("com.example.resources.RandomResource", mock(Meter.class)),
+            this.<Timer>map()
+        );
+
+        final ArgumentCaptor<InfluxDbPoint> influxDbPointCaptor = ArgumentCaptor.forClass(InfluxDbPoint.class);
+        Mockito.verify(influxDb, atLeastOnce()).appendPoints(influxDbPointCaptor.capture());
+        InfluxDbPoint point = influxDbPointCaptor.getValue();
+
+        assertThat(point.getMeasurement()).isEqualTo("resources");
```

Comment:
cool, check that it's tagged too! e.g. metricName=com.example.resources.RandomResource